Error Logging test flake

```
Chrome 56.0.2924 (Linux 0.0.0) Logging rethrowAsync should rethrow a single error FAILED
	AssertionError: expected 'intended _reported_' to equal 'intended'
	    at Context.<anonymous> (/tmp/ee8881a36732455386f0642b0f77b860.browserify:150303:32 <- /home/travis/build/ampproject/amphtml/test/functional/test-log.js:551:31)
```

Similar test flake
```
Chrome 45.0.2454 (Linux 0.0.0) Logging rethrowAsync should rethrow error with many messages FAILED
	AssertionError: expected 'first second third _reported_' to equal 'first second third'
	    at Context.<anonymous> (/tmp/b840555e97e699550ca85f6453e8421b.browserify:151367:32 <- /home/travis/build/ampproject/amphtml/test/functional/test-log.js:562:31)
```

--- a/test/functional/test-log.js
+++ b/test/functional/test-log.js
@@ -569,7 +569,7 @@ describe('Logging', () => {
       } catch (e) {
         error = e;
       }
-      expect(error.message).to.equal('first second third');
+      expect(error.message).to.match(/^first second third/);
     });
 
     it('should rethrow error with original error and messages', () => {

